xml/sle_update_preparation.xml

```diff
@@ -102,36 +97,41 @@
  <sect1 xml:id="sec-update-preparation-packagelist">
   <title>Listing installed packages and repositories</title>
   <para>
+   <remark>jufa: add more explanation why this list is needed. to easily restore for helping to set-up the new system? 
+    for reverting the upgrade? 2021-02-08</remark>
    You can save a list of installed packages, for example
    when doing a fresh install of a new major &slea; release or
    reverting to the old version.
   </para>
-  <para>
+  <note>
+   <para>
+    <remark>jufa: check if manual editing isstill relevant 2021-02-09</remark>
    Be aware that not all installed packages or used repositories are
    available in newer releases of &sle;. Some may have been renamed and
    others replaced. It is also possible that some packages are still
    available for legacy purposes while another package is used by
    default. Therefore some manual editing of the files might be
    necessary. This can be done with any text editor.
-  </para>
-  <para>
-   Create a file named <filename>repositories.bak.repo</filename> containing a
-   list of all used repositories:
-  </para>
-  <screen>&prompt.root;<command>zypper</command> lr -e repositories.bak</screen>
-  <para>
-   Also create a file named <filename>installed-software.bak</filename>
-   containing a list of all installed packages:
-  </para>
-  <screen>&prompt.root;<command>rpm</command> -qa --queryformat '%{NAME}\n' > installed-software.bak</screen>
-  <para>
+   </para>
+  </note>
+  
+  <procedure>
+   <step>
+    <para>Create a file named <filename>repositories.bak.repo</filename> containing a
+    list of all used repositories:</para>
+    <screen>&prompt.root;<command>zypper</command> lr -e repositories.bak</screen>
+   </step>
+   <step>
+    <para>Also create a file named <filename>installed-software.bak</filename>
+    containing a list of all installed packages:</para>
+    <screen>&prompt.root;<command>rpm</command> -qa --queryformat '%{NAME}\n' > installed-software.bak</screen>
+   </step>
+   <step><para>
```

Comment:
Please insert a line break between step and para

Reply:
fixed